proc/proc.go

```diff
@@ -106,6 +106,9 @@ func GetKernelModules(modulesPath string,
 		Size:    int(etext.Address - stext.Address),
 	})
 
+	atLeastOneValidAddress := false
+	lastInvalidAddressLine := ""
```

Comment:
`lastInvalidAddressLine` isn't used any more. Did you run a `make lint` locally?
```suggestion
```

Reply:
oups, sorry